Updates to CAO and e-sign process

TQuock opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

TQuock committed:
Updates to COA and e-sign process

it33 commented on operations/operations/company-agreements.md:
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
+The following staff are approved to send contracts to be initialed and e-signed
+
+* Lynn Conway  - primary for staff agreements and consulting agreements
+* Amy Nicole - primary for vendor contracts and corproate agreements 
```
1. Amy Nicol (no "e")
2. "corproate" typo?

TQuock replied:
```suggestion
* Amy Nicol - primary for vendor contracts and corporate agreements 
```

it33 commented on operations/operations/company-agreements.md:
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
+
+* Lynn Conway  - primary for staff agreements and consulting agreements
+* Amy Nicole - primary for vendor contracts and corproate agreements 
+* Kaely Wright - primary for sales agreements and customer supplier forms
```
Please check people's names in general for this section? "Kaely" doesn't seem right.

TQuock replied:
```suggestion
* Kaelyn Wright - primary for sales agreements and customer supplier forms
```

TQuock replied:
updated. Was a moving a little too quickly

it33 commented on operations/operations/company-agreements.md:
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
+* Natalie Jew \(HR\) - employment contracts, benefit contracts, vendor contracts (HR) 
+* Aneal Vallurupalli \(Finance\) - sales contracts, corporate contracts, vendor contracts
+* Tim Quock \(Accounting and Procurement\) - sales contracts, corporate contracts, vendor contracts
+* VP \([MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence#mattermost-leadership-team-mlt)\) - vendor contracts
```
I don't think MLT should be CAO. 

1. I'd prefer someone from G&A to initial the initial vendor agreement so we have clarity terms for things like indemnity, termination for convenience, jurisdiction, etc. are properly reviewed. 

2. I'm fine having MLT initial in addition to a CAO, for things like spend over $50K, etc. but we need someone trained in contract review as CAO.

TQuock replied:
```suggestion

```

it33 reviewed (dismissed):
Some changes requested. In addition, could I ask that in this Pull Request we also add something to this page noting that Pull Requests for updates need sign-off by both VP Finance & Ops and CEO in the PR process? 

What's laid out here is critical for keeping the integrity of our review process for company agreements.

TQuock commented on operations/operations/company-agreements.md:
```diff
@@ -6,27 +6,39 @@ Company agreements should only be signed by CEO, Ian Tien, after the approval of
+The following staff are approved to send contracts to be initialed and e-signed
+
+* Lynn Conway  - primary for staff agreements and consulting agreements
+* Amy Nicole - primary for vendor contracts and corproate agreements 
```
``
```suggestion
* Amy Nicol - primary for vendor contracts and corporate agreements 
```

TQuock reviewed (commented):
@it33 I have made the pull request https://github.com/TQuock/mattermost-handbook/pull/1#pullrequestreview-478136345

it33 commented:
Looks good to me, thanks Tim!

justinegeffen committed:
Update operations/operations/company-agreements.md

Co-authored-by: TQuock <52299088+TQuock@users.noreply.github.com>

justinegeffen committed:
Update operations/operations/company-agreements.md

Co-authored-by: TQuock <52299088+TQuock@users.noreply.github.com>

justinegeffen committed:
Merge branch '0.2.1' into 0.2.1

justinegeffen committed:
Update operations/operations/company-agreements.md

Co-authored-by: TQuock <52299088+TQuock@users.noreply.github.com>

justinegeffen committed:
Update company-agreements.md

Added note to page as per Ian's request. Formatting adjustments and removed trailing spaces.

justinegeffen commented:
> Some changes requested. In addition, could I ask that in this Pull Request we also add something to this page noting that Pull Requests for updates need sign-off by both VP Finance & Ops and CEO in the PR process?
> 
> What's laid out here is critical for keeping the integrity of our review process for company agreements.

I've updated the page with that note.

justinegeffen commented:
@TQuock, I've fixed the conflict. Please let me know whether anything else needs to be adjusted. Thanks! :)

TQuock commented:
@justinegeffen everything looks good from my end
